core/src/test/java/org/testcontainers/DockerClientFactoryTest.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers;
+
+import org.junit.Test;
+import org.testcontainers.dockerclient.LogToStringContainerCallback;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created mgorelikov on 05/04/2017
+ * Test for {@link DockerClientFactory}.
+ */
+public class DockerClientFactoryTest {
+
+  private static final String SUCCESS = "SUCCESS";
+
+  @Test
+  public void runCommandInsideDockerShouldPullImageIfItDoesNotExistsLocally() {
```

Comment:
I think this test does too much. What it should test that `runInsideDocker` does not fail if image wasn't pulled yet